todo_app/app.py

```diff
@@ -8,4 +9,11 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    return render_template('index.html', items = items)
+
+@app.route('/create-todo', methods=['Post'])
+def create_new_todo():
+    new_todo_title = request.form['todo-name']
+    add_item(new_todo_title)
+    return index()
```

Comment:
This is a super minor thing, but it's slightly nicer to return a `redirect('/')` (you'll need to import `redirect` from Flask.
This is so that the URL won't show "create-todo" and it will mean the browser will ahndle back and refreshes better

Again, this is super minor